sdk/textanalytics/ai-text-analytics/src/util.ts

```diff
@@ -54,3 +55,7 @@ export function findOpinionIndex(pointer: string): OpinionIndex {
     throw new Error(`Pointer "${pointer}" is not a valid opinion pointer`);
   }
 }
+
+export function addStrEncodingParam<T>(options: T): T & { stringIndexType: StringIndexType } {
+  return { ...options, stringIndexType: "Utf16CodeUnit" };
```

Comment:
@willmtemple I believe this is the right encoding but I wanted to double check.

Reply:
Yep, JS is a UTF-16 language.